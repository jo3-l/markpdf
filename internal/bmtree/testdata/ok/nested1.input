p1. topic 1
    p2. topic 1.1
    p3. topic 1.2