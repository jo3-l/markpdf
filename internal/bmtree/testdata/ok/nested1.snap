- "topic 1" @ p1
	- "topic 1.1" @ p2
	- "topic 1.2" @ p3